templates.go

```diff
@@ -626,6 +625,7 @@ coreos:
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/calico/client-crt.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/calico/client-crt.pem to be written' && sleep 1; done"
       ExecStartPre=/bin/bash -c "while [ ! -f /etc/kubernetes/ssl/calico/client-key.pem ]; do echo 'Waiting for /etc/kubernetes/ssl/calico/client-key.pem to be written' && sleep 1; done"
       ExecStartPre=/bin/bash -c "while ! curl --output /dev/null --silent --fail --cacert /etc/kubernetes/ssl/calico/client-ca.pem --cert /etc/kubernetes/ssl/calico/client-crt.pem --key /etc/kubernetes/ssl/calico/client-key.pem https://{{ .Cluster.Etcd.Domain }}:2379/version; do sleep 1 && echo 'Waiting for etcd master to be responsive'; done"
+      ExecStartPre=/bin/bash -c "source /etc/calico-environment"
```

Comment:
Network environment only has values so there is no evaluation issue.

```
cat /etc/network-environment
LO_IPV4=127.0.0.1
ETH0_IPV4=10.0.134.68
DEFAULT_IPV4=10.0.134.68
DOCKER0_IPV4=172.17.0.1
```

In the calico env file I need to set BRIDGE_IP to DEFAULT_IPV4 but the file isn't evaluated so it doesn't work.

```
cat /etc/calico-environment
# On AWS use internal IP as bridge IP
BRIDGE_IP=$DEFAULT_IPV4
```

It may be we need a better way to set this env var.